arcade/context.py

```diff
@@ -42,7 +42,7 @@ class ArcadeContext(Context):
 
     atlas_size = 512, 512
 
-    def __init__(self, window: pyglet.window.Window, gc_mode: str = "context_gc", gl_api: str = "gl"):
+    def __init__(self, window: pyglet.window.Window, gc_mode: str = "context_gc", gl_api: str = "gl") -> None:
```

Comment:
I thought `__init__` methods never needed a return type annotation.

Reply:
Well code automatically does it so I just thought I would do it on __init__ as well. Not sure about the convention for that though.

Reply:
It seems like it should be -> None.
Look at:
https://stackoverflow.com/questions/46779046/correct-type-annotation-for-init